### PR TITLE
[descheduler] migrate descheduler to distroless

### PR DESCRIPTION
**Pull request by artbin**
## Description

Descheduler are now based on distroless image.

## Why do we need it, and what problem does it solve?

We'd like to base our images on a distroless image.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: descheduler
type: feature
summary: Images are based on a distroless image.
impact_level: default
```

### CODE REVIEW

**Comment by artbin** [comment]
/e2e/abort openstack;Standard;containerd;1.25 6466612028 6466612028-1-con-1-25 /sys/deckhouse-oss/install:pr6105 debian@185.91.53.87

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @artbin. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @artbin. I've started the workflow for label 'e2e/run/openstack'...

**Review by RomanenkoDenys — COMMENTED** [review]
LGTM

File: modules/400-descheduler/images/descheduler/werf.inc.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{{- $version := "0.21.0" }}
+---
+artifact: {{ .ModuleName }}/build-artifact
+from: {{ .Images.BASE_GOLANG_20_ALPINE }}
+mount:
+  - fromPath: ~/go-pkg-cache
+    to: /go/pkg
+shell:
+  beforeInstall:
+    - apk add --no-cache git make
```
Comment (aleksey-su) [inline modules/400-descheduler/images/descheduler/werf.inc.yaml:10]: exclude make, not used

Comment (artbin) [reply modules/400-descheduler/images/descheduler/werf.inc.yaml]: done

**Review by name212 — APPROVED** [review]

**Pull request merged**